### DISCUSSION TITLE
Create durables not working as expected!?

### DISCUSSION THREAD

**Discussion by vellyr8x8** [postmanlabs/postman-flows#330] [Q&A] [answered]
Hey guys,
I've been trying to use the set durables block to store an **access_token** which I get from an access_token request. 
This token is available to me in response body as path **data/body/access_token**
ex:
<img width="1153" alt="Screenshot 2022-06-08 at 12 29 16 PM" src="https://user-images.githubusercontent.com/94951386/172552809-c51afa17-27b6-409e-8ecf-33f721b03eb7.png">

When I actually try to provide the value path for the access_token in durables block it limits me to set path up till **data/body**
instead of **data/body/access_token**
Here's an example:
<img width="851" alt="image" src="https://user-images.githubusercontent.com/94951386/172551642-0fff52db-ed2e-4b4e-8872-6d4e55ad8825.png">

Let me know if i'm doing something wrong here or theres an actual issue here. 
Thanks!

**Comment by Mr2000** [comment, marked as answer]
Hi @vellyr8x8 ,
From what I see, you should first define an Example for your access_token request. By doing so, Postman will be able to detect the content of your response (and all its fields)

When launching your request, in the response area just click on "Save Response" link (next to your response status) to generate an Example from your request

**Reply by vellyr8x8** [reply]
Thank you @Mr2000 !! This worked for me.
